docs/dev-manuals/adr/0001-use-graphql-api-gateway.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 2
+---
+
+# Use GraphQL API Gateway
+
+## Context and Problem Statement
+
+The frontend needs to be able to send requests to the backend.
+As such requests need to be instrumented with tracing metadata, and authentication needs to be performed, an API gateway should be provided.
+However, different API technologies exist, and we need to decide which one to use
+
+## Considered Options
+
+- GraphQL
+- REST
+- gRPC Web
+
+## Decision Outcome
+
+Chosen option: "GraphQL", because
+
+- allows frontend to use one unified API to fetch data from the backend
+
+### Consequences
+
+- Good, because allows frontend to easily fetch and combine data from different sources in the backend
+- Neutral, because requires us to find a way how to combine API of different services into one unified GraphQL API
+
+## Pros and Cons of the Options
+
+### GraphQL
+
+- Good, because allows frontend to easily fetch and combine data from different sources in the backend
+- Good, because built-in strict type system
+- Good, because GraphiQL allows users to explore the API, including with a graphical interface to build requests
```

Comment:
Typo in "GraphiQL"

Reply:
What is the typo? Am I just too blind to see it?

Reply:
Never mind, I thought you just meant GraphQL without the i and I just didn't know that GraphiQL existed. My bad.